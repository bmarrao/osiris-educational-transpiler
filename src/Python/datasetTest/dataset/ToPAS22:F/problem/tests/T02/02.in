10
3
4 5 7
3
4 -1 5
2
0 0
3
4 -1 3
4
4 5 -1 6
3
-1 3 -1
4
-1 -1 1 -1
3
-1 1 -1
10
-1 -1 -1 -1 -1 -1 5 -1 -1 -1
1
0
