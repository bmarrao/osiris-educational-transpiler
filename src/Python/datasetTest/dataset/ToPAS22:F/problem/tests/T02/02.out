Impossivel
Impossivel
Impossivel
Impossivel
Impossivel
Possivel
Impossivel
Possivel
Impossivel
Possivel
